tasks/reboot.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+- name: reboot the machine
```

Comment:
this should rather be a handler.


https://docs.ansible.com/ansible/latest/user_guide/playbooks_handlers.html

This way it would be ran only if reboot var set, and something changed.

Reply:
See as an example: https://github.com/geerlingguy/ansible-role-haproxy/blob/master/handlers/main.yml